### PR TITLE
Adds 'Mods only' visualizer

### PR DESCRIPTION
**Pull request by colinta**
I made this for my own purposes, but I'll offer it up in the spirit of open source.  Based on the Svelte visualizer, this one displays the current modifiers and nothing more.

I use it because I have a custom keyboard that implements sticky keys, and I needed to see them on screen (it's easy to lose track of which keys are pressed).

Not really useful for screencasting, but it does show how to create a new visualizer. 🤷‍♂️ 

Example of pressing "command+shift" and "command+control+alt+shift".

Also, Svelte has a blue background... why?  It's ugly.  I can fix it with another PR if this design was unintentional.

screen shot 2019-02-27 at 11 08 05 am
screen shot 2019-02-27 at 11 07 47 am

### CODE REVIEW

File: keycastr/ModsVisualizer.m [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+	[_visualizerWindow setMovableByWindowBackground:YES];
+	[_visualizerWindow setFrame:r display:NO];
+	[_visualizerWindow setFrameAutosaveName:@"svelte visualizerFrame"];
+	[_visualizerWindow setFrameUsingName:@"svelte visualizerFrame"];
+	[_visualizerWindow setOpaque:NO];
+
+	_visualizerView = [[ModsVisualizerView alloc] initWithFrame:r];
+	[_visualizerWindow setContentView:_visualizerView];
+
+    _displayAll = [[[NSUserDefaults standardUserDefaults] valueForKey:@"svelte.displayAll"] boolValue];
```
Comment (colinta) [inline keycastr/ModsVisualizer.m]: oops, I don't need this, I'll remove.

**Comment by akitchen** [comment]
Hey @colinta thank you so much for the pull request! KeyCastr has been very much on the back burner recently (as you can probably tell) but I'll try to take a closer look at this soon.

**Comment by akitchen** [comment]
Hey @colinta I got around to looking at this yesterday. Great work! I'd like to figure out a way to introduce this into KeyCastr somehow, perhaps as an extension of the Svelte visualizer or as its own thing. I'll try to follow up with more specific comments/suggestions soon.

> Also, Svelte has a blue background... why? It's ugly. I can fix it with another PR if this design was unintentional.

I noticed that when building your code, but not in the currently released version. It's in no way intentional. We'll have to dig in and fix that before the next release one way or another if it's not specific to your branch.

**Comment by akitchen** [comment]
Looks like this was closed automatically when I renamed the default branch to main. Please re-target against `main` and reopen.

**Comment by colinta** [comment]
Done!

**Pull request closed without merging**